grouper/src/grouper/edu/internet2/middleware/grouper/Group.java

```diff
@@ -31,6 +31,20 @@
 */
 
 package edu.internet2.middleware.grouper;
+import static edu.internet2.middleware.grouper.hooks.examples.GroupTypeTupleIncludeExcludeHook.excludeDescription;

```

Comment:
Wouldnt you just want to do:

import static edu.internet2.middleware.grouper.hooks.examples.GroupTypeTupleIncludeExcludeHook.* ?